pkg/network/tracer/event.go

```diff
@@ -376,6 +390,30 @@ func newIPRouteDest(source, dest util.Address, netns uint32) *ipRouteDest {
 	return d
 }
 
+func getPortType(p uint16) network.EphemeralPortType {
```

Comment:
nit: `getPortType` is a bit ambiguous in regards to what the type you will return. Maybe `isEphemeral` would help readability?